### PR TITLE
[vtctl] Deprecate Throttler RPCs

### LINKED ISSUE #9960: Deprecate Throttler vtctl commands
The following are used for the old `SplitClone` and `VerticalSplitClone` workflows. The new VReplication workflows (`Reshard`, `MoveTables`, and friends) use freno for throttling.

This issue covers the following commands:
- `ThrottlerMaxRates`
- `ThrottlerSetMaxRate`
- `GetThrottlerConfiguration`
- `UpdateThrottlerConfiguration`
- `ResetThrottlerConfiguration`

We will delete these, and the associated vtctl/wrangler code after the next release.

### PR DESCRIPTION
**Pull request by ajm188**
## Description

The new VReplication flows do not rely on these, which means we can delete them after vtworker is gone.

### Demo

```
➜  local git:(andrew/vtctldserver-deprecate-throttler) ✗ vtctlclient --server ":15999" help | grep Throttler                  
Resharding Throttler:
  (DEPRECATED) ThrottlerMaxRates -server <vtworker or vttablet>
  (DEPRECATED) ThrottlerSetMaxRate -server <vtworker or vttablet> <rate>
  (DEPRECATED) GetThrottlerConfiguration -server <vtworker or vttablet> [<throttler name>]
  (DEPRECATED) UpdateThrottlerConfiguration -server <vtworker or vttablet> [-copy_zero_values] "<configuration protobuf text>" [<throttler name>]
  (DEPRECATED) ResetThrottlerConfiguration -server <vtworker or vttablet> [<throttler name>]
➜  local git:(andrew/vtctldserver-deprecate-throttler) ✗ vtctlclient --server ":15999" ThrottlerMaxRates -- -h
WARNING: ThrottlerMaxRates is deprecated and will be removed in a future release. Use the new Reshard/MoveTables workflows instead.
Usage: ThrottlerMaxRates -server <vtworker or vttablet>

Returns the current max rate of all active resharding throttlers on the server.

  --server string
        vtworker or vttablet to connect to
➜  local git:(andrew/vtctldserver-deprecate-throttler) ✗ 
```

## Related Issue(s)

Closes #9960

## Checklist
- [x] Should this PR be backported? **no**
- [x] Tests were added or are not required **n/a**
- [x] Documentation was added or is not required

### CODE REVIEW

**Review by GuptaManan100 — APPROVED** [review]
We should wait for @deepthi's approval too

**Review by deepthi — APPROVED** [review]

**Pull request merged**
